pyodide_build/pywasmcross.py

```diff
@@ -270,9 +279,19 @@ def handle_command(line, args, dryrun=False):
         if arg.startswith("-L/usr"):
             continue
         if arg.startswith("-l"):
-            if arg[2:] in replace_libs:
-                arg = "-l" + replace_libs[arg[2:]]
-
+            # WASM link doesn't like libraries being included twice
+            # skip second one
+            if arg in used_libs:
+                continue
+            used_libs[arg] = 1
+        if arg.startswith("-l"):
+            for lib_name in replace_libs.keys():
+                # this enables glob style **/* matching
+                if PurePosixPath(arg[2:]).match(lib_name):
```

Comment:
This is a useful change but I don't think it is required for the emscripten upgrade?